pgrx/src/trigger_support/pg_trigger.rs

```diff
@@ -17,13 +17,11 @@ use crate::trigger_support::{
 };
 use std::ffi::c_char;
 
-/**
-The datatype accepted by a trigger
-
-A safe structure providing the an API similar to the constants provided in a PL/pgSQL function.
-
-Usage examples exist in the module level docs.
-*/
```

Comment:
hm. there are a lot of block doc comments in this repo.

Reply:
oh, oops, i began hacking on this using v0.11, this was a merge issue, reverted

Reply:
Ah, that makes sense.